src/static/riot/search.tag

```diff
@@ -813,6 +834,18 @@
             display block
             @media screen and (min-width 500px)
                 display inline-block
+
+        @media screen and (min-width 1921px)
+            *
+                font-size 1.15rem !important
```

Comment:
Whoa, does this mess up anything else? Seems like a very broad change, but I suppose this is how you have to do it to make text larger... makes me feel a bit weird but probably fine

Reply:
I believe this is scoped within the competition tile, so it doesn't effect fonts outside of that element. I had to add font-size changes on header elements to increase readability, for example.